### ISSUE TITLE
Failed to compile with VS2022+UE5.02

### ISSUE THREAD

**Issue by ArcEcho** [GPUOpen-Effects/FidelityFX-FSR2#14]
![image](https://user-images.githubusercontent.com/7297514/175856081-0f8e8793-479a-4e18-88de-47c4775a06ea.png)

File: FSR2\Source\ffx-fsr2-api\dx12\shaders\ffx_fsr2_shaders_dx12.cpp Line: 391

![image](https://user-images.githubusercontent.com/7297514/175856496-243a4484-37c5-4889-b0df-a1b3b88738e1.png)

```c++
//memset(&outBlob, 0, sizeof(Fsr2ShaderBlobDX12));
memset(outBlob, 0, sizeof(Fsr2ShaderBlobDX12));
```

**Comment by TheNewJavaman** [comment]
I also encountered this bug today, fixed using the same snippet ^

**Comment by TheNewJavaman** [comment]
Will be fixed in the next release by #33 and #40 (same change, different PRs) according an official AMD dev
